### PR TITLE
Fix for issue #29

### PR DESCRIPTION
**Pull request by stefanuebe**
Contains fix for https://github.com/vaadin/incubator-tooltip/issues/29

@jtreml will test the contents today/tomorrow and give you feedback if this may be merged from his point of view.

### CODE REVIEW

**Comment by jtreml** [comment]
This indeed fixes the positioning problem in the original application where #29 was discovered.

The scrolling-adjustment does not work, though; In the original application it's used inside a div that scrolls rather than the entire page being scrolled. Code uses `window.pageYOffset` for calculations which is likely why it doesn't work in that case.

Plus some general edge cases related to scrolling:
tooltip-issues-scrolling

In summary, we'd be breaking edge case behavior (tooltip shown while scrolling) in order to fix a probably somewhat more common issues (tooltip used in absolutely/relatively positioned container).
